### DISCUSSION TITLE
What node version, commit, etc should I try and use (Ubuntu 22.04)

### DISCUSSION THREAD

**Discussion by zlacelle** [google/schedviz#53] [General]
Hello,

I'm trying to get this running in Ubuntu 22.04, following the instructions for yarn and nodejs. I will admit I use bazel/node very little. Seems like I need nodejs >= 14, so I already have to use a different package source (which is fine). Should I use 16? 18? Which is the most tested version with this project?

Additionally, is there a particularly stable commit or release I should try?

Currently,  I get lots of different errors trying to run, so before I report them, I'd like to start at "known stable"

**Comment by sabarabc** [comment]
It was working as of 0beae2e472bec9e6cc5a4754f37eb68257eda8fe

As time has gone on, dependencies have changed and this project is no longer actively maintained.

**Comment by glytchninja** [comment]
How can we keep it alive?

**Comment by ilhamster** [comment]
Thanks for asking.

Given the tool's status internally (still in use and maintained but not in
active development) and the difficulty we've had keeping the internal and
external versions in sync, I think we'd be OK with the external version
diverging, especially if it gets fixed.  I'll be happy to approve PRs that
fix it, and would also be happy to add a link to a fixed fork.
